modules/update-service-configure-cvo.adoc

```diff
@@ -0,0 +1,38 @@
+[id="update-service-configure-cvo_{context}"]
+
+.Prerequisites
+
```

Comment:
The current and target releases have been mirrored to the internal registry is also a prerequisite.

Reply:
done